craft_store/models/_common_list_releases_model.py

```diff
@@ -41,8 +41,8 @@ class ChannelsModel(MarshableModel):
     :param track: the channel track.
     """
 
-    branch: str | None
-    fallback: str | None
+    branch: str | None = None
+    fallback: str | None = None
```

Comment:
Shouldn't we put them at the end if they have default value to make it compliant with dataclasses?

Reply:
pydantic doesn't have the same rules as dataclasses. In pydantic 1, `str | None` was the same as `Optional[str]`, which meant it had a default value of `None` anyway, so this isn't actually changing the defaults.